libs/ml/include/ml/charge_estimation/ops/constants.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2020 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "ml/charge_estimation/types.hpp"
+
+namespace fetch {
+namespace ml {
+namespace ops {
```

Comment:
the namespace and the file directory are contrary here: i.e. charge_estimation/ops/ vs ops{ charge_cost{
this is a little unintuitive. Perhaps the namespaces for charge_cost and ops should be the other way around?

Reply:
Agreed, done